packages/common/src/ui/components/Button.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Button as MuiButton, ButtonProps, Theme } from '@material-ui/core';
+import { styled } from '@material-ui/styles';
+
+const StyledButton = styled(MuiButton)(({ theme }: { theme: Theme }) => ({
+  backgroundColor: '#fff',
+  borderRadius: 24,
+  color: theme.palette.primary.main,
+  fontWeight: 'bold',
+  height: 40,
+  marginRight: 10,
+  minWidth: 115,
+  textTransform: 'none',
+  boxShadow: theme.shadows[1],
+}));
+
+const Button: React.FC<ButtonProps> = props => {
```

Comment:
Wondering if we have a wide range of buttons if the label should be more specific?

I am thinking of separate buttons for having an icon and not (rather than props) and other things to avoid bloated components with a bunch of props and conditions defining behaviour

Reply:
agree - and partly held off because I didn't want to name the buttons 🙃